spec/javascripts/search_spec.js

```diff
@@ -1,97 +1,94 @@
 describe('GOVUK.search', function () {
-  var $results;
+  var $results
 
   beforeEach(function () {
-    $results = $('<div id="results"></div>');
-    $('body').append($results);
-  });
+    $results = $('<div id="results"></div>')
+    $('body').append($results)
+  })
 
   afterEach(function () {
-    $results.remove();
-  });
+    $results.remove()
+  })
 
   describe('extractSearchURLs', function () {
-    var $resultsList;
+    var $resultsList
 
     describe('no results found', function () {
       beforeEach(function () {
-        GOVUK.analytics = GOVUK.analytics || {};
-        GOVUK.analytics.trackEvent = GOVUK.analytics.trackEvent || function(args) {};
-        spyOn(GOVUK.analytics, 'trackEvent');
+        GOVUK.analytics = GOVUK.analytics || {}
+        GOVUK.analytics.trackEvent = GOVUK.analytics.trackEvent || function (args) {}
+        spyOn(GOVUK.analytics, 'trackEvent')
         $resultsList = []
-        $results.append($resultsList);
-      });
+        $results.append($resultsList)
+      })
 
       it('returns an empty array if no results found', function () {
-        expect(GOVUK.search.extractSearchURLs($resultsList)).toEqual([]);
-      });
+        expect(GOVUK.search.extractSearchURLs($resultsList)).toEqual([])
+      })
 
       it('does not fire an event when trackSearchResultsAndSuggestions is called', function () {
-        var extractedURLs = GOVUK.search.extractSearchURLs($resultsList);
-        var $searchResults = $('#results .results-list');
-        GOVUK.search.trackSearchResultsAndSuggestions($searchResults);
-        expect(GOVUK.analytics.trackEvent).not.toHaveBeenCalled();
-      });
+        var $searchResults = $('#results .results-list')
+        GOVUK.search.trackSearchResultsAndSuggestions($searchResults)
+        expect(GOVUK.analytics.trackEvent).not.toHaveBeenCalled()
+      })
 
       describe('suggestions present', function () {
         beforeEach(function () {
           var $suggestion = $('<fieldset class="spelling-suggestion">' +
                               '<p>Did you mean ' +
                               '<a href="/search?o=testin&amp;q=testing">testing</a>' +
                               '</p>' +
-                              '</fieldset>');
-          $results.append($suggestion);
-        });
+                              '</fieldset>')
+          $results.append($suggestion)
+        })
 
         it('assigns the suggestions key when suggestions are present', function () {
           expect(GOVUK.search.buildSearchResultsData($resultsList))
             .toEqual({
               'urls': [],
               'suggestion': 'testing'
-          });
-        });
+            })
```

Comment:
Nitpick: is this indentation correct?

Reply:
Yep - matches up to the `.toEqual` on line 47. In any case, this is what `standard --fix` did for me.